integrations/instagram/src/misc/utils.ts

```diff
@@ -62,18 +71,43 @@ export function getCarouselMessage(payload: Carousel): MessengerTypes.Attachment
   }
 }
 
-export function getChoiceMessage(payload: Choice | Dropdown): MessengerTypes.AttachmentMessage {
+export function getChoiceMessage(payload: Choice | Dropdown): MessengerTypes.TextMessage {
+  if (!payload.options.length) {
+    return { text: payload.text }
+  }
+
+  if (payload.options.length > 13) {
+    return {
+      text: `${payload.text}\n\n${payload.options.map((o, idx) => `${idx + 1}. ${o.label}`).join('\n')}`,
```

Comment:
Is there a particular reason we want to use this non-interactive approach for the bot user when the number of choices exceeds the limit? If possible, it would be preferable to send multiple-choice message groups if the number of choices provided exceeds the limit. The Whatsapp integration does this if you want to check it for reference.

Reply:
I've reused what has been done for the Messenger integration here, is this a case that actually happens a lot ?

Reply:
I'm not aware if people have Instagram/Messengers bots with so many options, but yeah, I'd argue it's uncommon and probably not the best UX for the user anyway even if were splitting them in choice groups. I think we can leave this as-is for now.